transform/mattermost-analytics/models/intermediate/hightouch/int_customers_with_onprem_subs.sql

```diff
@@ -0,0 +1,188 @@
+{{ config
+    ({
+        "materialized": 'table',
+        "schema": "int_hightouch",
+        "unique_key":['id'],
+  })
+}}
+
+WITH latest_credit_card_address AS (
```

Comment:
should we keep the old name, now that we don't source from the `credit_cards` source anymore? Also, it looks like this is being used in other models (e.g. `customers_with_cloud_subs`), can we factor it out?

Reply:
Dropped this model completely.